src/pipx/paths.py

```diff
@@ -77,17 +93,43 @@ def make_local(self) -> None:
         self._base_home = get_expanded_environ("PIPX_HOME")
         self._base_bin = get_expanded_environ("PIPX_BIN_DIR")
         self._base_man = get_expanded_environ("PIPX_MAN_DIR")
-        self._home_exists = self._base_home is not None or self._fallback_home.exists()
+        self._home_exists = self._base_home is not None or any(fallback.exists() for fallback in self._fallback_homes)
 
     def make_global(self) -> None:
         self._base_home = get_expanded_environ("PIPX_GLOBAL_HOME") or DEFAULT_PIPX_GLOBAL_HOME
         self._base_bin = get_expanded_environ("PIPX_GLOBAL_BIN_DIR") or DEFAULT_PIPX_GLOBAL_BIN_DIR
         self._base_man = get_expanded_environ("PIPX_GLOBAL_MAN_DIR") or DEFAULT_PIPX_GLOBAL_MAN_DIR
-        self._home_exists = self._base_home is not None or self._fallback_home.exists()
+        self._home_exists = self._base_home is not None or any(fallback.exists() for fallback in self._fallback_homes)
 
     @property
     def standalone_python_cachedir(self) -> Path:
         return self.home / "py"
 
+    def log_warnings(self):
+        if " " in str(self.home):
+            logger.warning(
+                pipx_wrap(
+                    (
+                        ":hazard: Found a space in the home path. We heavily discourage this, due to "
+                        "multiple incompatibilities. Please check our docs for more information on this, "
+                        "as well as some pointers on how to migrate to a different home path."
+                    ),
+                    subsequent_indent=" " * 4,
+                )
+            )
+
+        if self._fallback_homes and DEFAULT_PIPX_HOME.exists():
```

Comment:
Could also be that `PIPX_HOME` has been set? We should check if `_PathContext.home` exists.